src/tests/unit/doughs-dao.test.js

```diff
@@ -61,17 +59,19 @@ describe('test doughs dao', () => {
   let connectionStub;
   let doughsDao;
   beforeEach(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
     serializeDoughsStub = sinon.stub().returns(baseGetDoughsReturn);
     serializeDoughStub = sinon.stub().returns(baseGetDoughsReturn);
-    afterEach(() => {
-      sinon.restore();
-    });
-    describe('getDoughs', () => {
-      beforeEach(() => {
-        connectionStub = sinon.stub().returns(executeReturn);
+    connectionStub = sinon.stub().returns(executeReturn);
+  });
+  afterEach(() => {
+    sinon.restore();
+  });
+  describe('getDoughs', () => {
+    beforeEach(() => {
+      connectionStub = sinon.stub().returns(executeReturn);
 
-        doughsDao = proxyquireDoughsDao(connectionStub, serializeDoughsStub, serializeDoughStub);
-      });
+      doughsDao = proxyquireDoughsDao(connectionStub, serializeDoughsStub, serializeDoughStub);
```

Comment:
The changes in this section made for a really messy rebase. I've verified that it works though and the final changes look OK to me.